lib/pages/discover.dart

```diff
@@ -103,12 +96,17 @@ class Device extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return GestureDetector(
-      child: Text(
-        ipAddr,
-        style: TextStyle(
-          color: Sinix.textColor,
-          fontSize: 18,
-        ),
+      child: Row(
+        children: <Widget>[
+          Icon(FeatherIcons.cast),
```

Comment:
Add some space between the icon and the IP address

Reply:
@sanket143 done, I have added a `SizedBox`